# HG changeset patch
# User Rik <rik@octave.org>
# Date 1444254099 25200
#      Wed Oct 07 14:41:39 2015 -0700
# Branch stable
# Node ID 1b62fc4e1b2f7a51982cd4582b49be4644726b44
# Parent  c6059134f5d34da8791f3570d37bd3fbf0ae83db
doc: Cuddle parentheses in example code of for loop.

* stmt.txi: Cuddle parentheses in example code of for loop.

diff --git a/doc/interpreter/stmt.txi b/doc/interpreter/stmt.txi
--- a/doc/interpreter/stmt.txi
+++ b/doc/interpreter/stmt.txi
@@ -536,17 +536,17 @@ executed.
 The following example shows another way to create a vector containing
 the first ten elements of the Fibonacci sequence, this time using the
 @code{for} statement:
 
 @example
 @group
 fib = ones (1, 10);
 for i = 3:10
-  fib (i) = fib (i-1) + fib (i-2);
+  fib(i) = fib(i-1) + fib(i-2);
 endfor
 @end group
 @end example
 
 @noindent
 This code works by first evaluating the expression @code{3:10}, to
 produce a range of values from 3 to 10 inclusive.  Then the variable
 @code{i} is assigned the first element of the range and the body of the
